content/applications/finance/accounting/taxes/avatax/avalara_portal.rst

```diff
@@ -0,0 +1,182 @@
+
+.. _avatax_portal:
+
+=======================
+Avalara (Avatax) portal
+=======================
+
+Avalara's (*AvaTax*) management console offers account management options including viewing/editing
+the transactions sent from Odoo to *AvaTax*, details on how the taxes are calculated, tax reporting,
+tax exemption management, and tax return resources.
+
+To access the console, first, navigate to either Avalara's `sandbox
+<https://sandbox.admin.avalara.com/>`_ or `production <https://admin.avalara.com/>`_ environment.
+This will depend on which type of account was set in the :doc:`integration <../avatax>`.
+
+.. _avalara_portal/transactions:
+
+Transactions
+============
+
+Next, to access transactions, click in the :guilabel:`Transactions` link on the main dashboard upon
+logging into the :ref:`console <avatax_portal>`. To manually access the *Transactions* page, while
+logged into the Avalara console, navigate to :menuselection:`Transactions --> Transactions`.
+
+Edit transaction
+----------------
+
+Click into a transaction to reveal more details about the transaction. These include sections on
+:guilabel:`Invoice detail`, :guilabel:`Additional info`, and :guilabel:`Customer info`. Click
+:icon:`fa-pencil` :guilabel:`Edit document details` to make changes to the transaction.
+
+A :guilabel:`Discount` can be added to adjust the invoice. This is especially useful in cases where
+the transaction has already synced with Avalara / *AvaTax* and changes need to be made afterward.
+
+.. _avalara_portal/filter:
+
+Filter
+------
+
+Filter transactions by setting the :guilabel:`From` and :guilabel:`To` fields, and configuring other
+fields to filter by, including:
+
+ - :guilabel:`Document Status`
+ - :guilabel:`Document Code`
+ - :guilabel:`Customer/Vendor Code`
+ - :guilabel:`Country`
+ - :guilabel:`Region`
+
+Sort by
+-------
+
+On the *Transactions* page, transactions will be listed below according to the set
+:ref:`avalara_portal/filter`, in the top half of the page. The following columns are available by
+default, to sort by ascending or descending order:
+
+ - :guilabel:`Doc Code`
+ - :guilabel:`Doc Date`
+ - :guilabel:`Doc Status`
+ - :guilabel:`Cust/Vendor Code`
+ - :guilabel:`Region`
+ - :guilabel:`Amount`
+ - :guilabel:`Not Taxed`
+ - :guilabel:`Tax`
```

Comment:
I seea "Taxable' field by defaut- might want to check if that's a default view or not.